releases/v0.25.md

```diff
@@ -0,0 +1,133 @@
+- (server) fix: webhook race condition on hook creation [#1175](https://github.com/go-vela/server/commit/89b0a4a3041a78087255f45529fef0be99110aac) - thanks [@jbrockopp](https://github.com/jbrockopp)!
+- (types) fix(swagger): remove old model annotations [#385](https://github.com/go-vela/types/commit/75a480864eca3de766d2a666db0938bd9d02a2e8) - thanks [@wass3r](https://github.com/wass3r)!
+- (types) fix(yaml): remediate insecure/archived buildkite lib in favor of go-yaml v3 [#386](https://github.com/go-vela/types/commit/76a66e72d5dcba850471be2b350dc28aec8e9d45) - thanks [@ecrupper](https://github.com/ecrupper)!
+- (types) fix: update doc links [#388](https://github.com/go-vela/types/commit/4dbb9a5a70f24da7a339907766ea1581395d1318) - thanks [@wass3r](https://github.com/wass3r)!
+- (ui) fix(resources): get all build resources [#811](https://github.com/go-vela/ui/commit/828488762031de52b1aa82f1ed1bd20051ab3d35) - thanks [@plyr4](https://github.com/plyr4)!
+- (ui) fix(secrets): clear form on update [#820](https://github.com/go-vela/ui/commit/9311d31df12fc3d8e791e063135ce25a686d59de) - thanks [@plyr4](https://github.com/plyr4)!
+- (ui) fix(viz): always fetch graph when clicking refresh [#821](https://github.com/go-vela/ui/commit/6b5a336d5767c07beb9a6284c4d4db33e8e12a17) - thanks [@plyr4](https://github.com/plyr4)!
+- (ui) fix: allow starlark limit to be customized [#814](https://github.com/go-vela/ui/commit/1459f6130265292ed8e18253aa78c75d0e5ddb77) - thanks [@plyr4](https://github.com/plyr4)!
+- (ui) fix: reduce excessive API calls [#810](https://github.com/go-vela/ui/commit/b83bba0661c7f2d4f5efc995c303cfd38fc3470d) - thanks [@wass3r](https://github.com/wass3r)!
+- (worker) fix(tests): pass context into Compile [#598](https://github.com/go-vela/worker/commit/a26112c6272da4faaeeab3e414356e4c5cc4bc62) - thanks [@plyr4](https://github.com/plyr4)!
+
+### ♻️ Refactors
+
+- (server) refactor(hook): nest API response and use Gorm preload to populate repo and build data [#1147](https://github.com/go-vela/server/commit/ddee4448143fef4d55d9b3946fb7c764248f843e) - thanks [@ecrupper](https://github.com/ecrupper)!
+- (server) refactor: use builtin min/max funcs [#1148](https://github.com/go-vela/server/commit/563ff5265f5e179b03d3867135d286b5fde1e125) - thanks [@wass3rw3rk](https://github.com/wass3rw3rk)!
+
+## 🔗 Release Links
+
+- https://github.com/go-vela/cli/releases
+- https://github.com/go-vela/sdk-go/releases
+- https://github.com/go-vela/server/releases
+- https://github.com/go-vela/types/releases
+- https://github.com/go-vela/ui/releases
+- https://github.com/go-vela/worker/releases
+
+## 💟 Thank you to all the contributors in this release!
+
+- @ecrupper
+- @jbrockopp
+- @JordanSussman
+- @KellyMerrick
+- @plyr4
+- @wass3r
+- @wass3rw3rk
+- @wsan3
+
+## Endpoint Changes
+
+TODO: UPDATE THIS SECTION
+
+### Endpoints with updated paths:
+
+TODO: UPDATE THIS SECTION
+
+### Endpoints with modified response bodies:
+
+TODO: UPDATE THIS SECTION
```

Comment:
remove since we callout at the top?

Reply:
sure, fwiw v24 had this additional section at the bottom
https://github.com/go-vela/community/blob/main/releases/v0.24.md#endpoint-changes

Reply:
removed most of them but kept endpoint-changes to be consistent with v24

Reply:
sounds good. i looked at 24 and totally missed that section :shame: